app/lib/services.rb

```diff
@@ -19,7 +19,8 @@ def self.rummager
 
   def self.email_alert_api
     @email_alert_api ||= GdsApi::EmailAlertApi.new(
-      Plek.find("email-alert-api")
+      Plek.current.find("email-alert-api"),
+      bearer_token: ENV["EMAIL_ALERT_API_BEARER_TOKEN"] || "wubbalubbadubdub"
```

Comment:
ha love the wubbalubbadubdub 😀 

May just be a personal preference but I tend to find `ENV.fetch("EMAIL_ALERT_API_BEARER_TOKEN", "wubbalubbadubdub")` is a bit easier on the eyes, what do you think?

Reply:
Updated in c0f8f41. Your eye ease is always paramount 😆